test/index.ts

```diff
@@ -63,7 +62,6 @@ const fakePaginator = {
         return;
       }
 
-      methods = arrify(methods);
```

Comment:
If I read this correctly it will turn methods into an array of arrays, doesn't look like this change had any impact but just want to double check.

Reply:
it won't change it into an array of arrays. `arrify(["hello"]) = ["hello"]` https://github.com/sindresorhus/arrify